crates/proposal-signing-backends/src/dkg.rs

```diff
@@ -96,73 +106,38 @@ impl super::ProposalSigningBackend for DkgProposalSigningBackend {
         tracing::debug!(
             nonce = %hex::encode(nonce.encode()),
             resource_id = %hex::encode(resource_id.into_bytes()),
-            src_chain_id = ?src_chain_id,
+            src_chain_id = ?self.typed_chain_id,
             proposal = %hex::encode(proposal.to_vec()),
             "sending proposal to DKG runtime"
         );
+
         let xt = tx_api.acknowledge_proposal(
             nonce,
             src_chain_id,
             ResourceId(resource_id.into_bytes()),
             BoundedVec(proposal.to_vec()),
         );
+        // webb dynamic payload
+        let execute_proposal_tx = WebbDynamicTxPayload {
+            pallet_name: Cow::Borrowed("dkgProposals"),
+            call_name: Cow::Borrowed("acknowledge_proposal"),
+            fields: vec![
+                Value::u128(u128::from(self.typed_chain_id.chain_id())),
+                Value::from_bytes(xt.call_data().encode()),
+            ],
+        };
+        let data_hash = utils::keccak256(xt.call_data().encode());
+        let tx_key = SledQueueKey::from_substrate_with_custom_key(
+            self.typed_chain_id.underlying_chain_id(),
+            make_execute_proposal_key(data_hash),
```

Comment:
function name here is misleading, there is not execute proposal in the process, I would suggest renaming it to something like:
```suggestion
            make_ acknowledge_proposal_key(data_hash),
```